platform-sdk/swirlds-common/src/main/java/com/swirlds/common/config/EventConfig.java

```diff
@@ -71,5 +75,6 @@ public record EventConfig(
         @ConfigProperty(defaultValue = "5") long eventsLogPeriod,
         @ConfigProperty(defaultValue = "/opt/hgcapp/eventsStreams") String eventsLogDir,
         @ConfigProperty(defaultValue = "true") boolean enableEventStreaming,
+        @ConfigProperty(defaultValue = "false") boolean useLegacyPrehandle,
```

Comment:
are we sure we want another feature flag?

Reply:
Talked it over with Austin. We agreed that all of the new intake pipeline stuff can use the single feature flag. Either use all of the new code, or none of it. Removed.